src/components/Cluster/ClusterDetail/AddNodePool.js

```diff
@@ -688,12 +688,6 @@ class AddNodePool extends Component {
           <StyledInput
             labelId={`scaling-range-${this.props.id}`}
             label='Scaling range'
-            hint={
```

Comment:
This was duplicated in NodeCountSelector, but hidden when in a certain context (see: https://github.com/giantswarm/happa/pull/1617/files#diff-d8b088a074d65e4a00a05a47c1687d4fL176). 

Instead of doing it twice I just allow the one in NodeCountSelector to show.

Reply:
Good find! We should avoid these "unspecified" css selectors in the future